app/models/invitiation_message_builder.rb

```diff
@@ -28,9 +28,9 @@ def error_message
 
   def success_message
     if successes? && no_email?
-      'An invitation has been created for: ' + successes.join(', ')
+      "An invitation has been created for: " + successes.join(', ')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.